features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43012/HCIDriver.cpp

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright (c) 2018 ARM Limited
+ *
```

Comment:
Can you add SPDX identifier (look at platform code for instance) here or to any new file? (follows our license guidelines)

Reply:
The copyright headers are defined by our legal department.
I think the SPDX identifier can be added with consecutive pull requests, once approved.
Please confirm this is not gating the initial PR.

Reply:
The SPDX identifier is already approved internally, will include with the next rebase (plan to get complete greentea test suite working later this week and submit the update).